### PR TITLE
pass DNT to pip service and sanitize error messages

### PR DESCRIPTION
**Pull request by trescube**
passes `dnt` header to PIP service when requested at API by caller.  Also sanitizes identifiable parameters from error messages.

### CODE REVIEW

File: service/pointinpolygon.js [hunk 1]
```diff
@@ -2,14 +2,51 @@ const logger = require( 'pelias-logger' ).get( 'pointinpolygon' );
 const request = require('request');
 const _ = require('lodash');
 
+function sanitizeUrl(requestUrl) {
+  return requestUrl.replace(/^(.+)\/.+\/.+$/, (match, base) => {
+    return `${base}/[removed]/[removed]`;
+  });
+}
```
Comment (missinglink) [inline service/pointinpolygon.js]: no tests for `sanitizeUrl()`, regex's always have edge cases?

Comment (trescube) [inline service/pointinpolygon.js]: There are tests [here](https://github.com/pelias/api/pull/852/files#diff-f1db4ccb690ad5f0b8838fc6aece9abbR213) and [here](https://github.com/pelias/api/pull/852/files#diff-f1db4ccb690ad5f0b8838fc6aece9abbR298).

Comment (missinglink) [inline service/pointinpolygon.js]: oh, missed those sorry
